cli/command/task_command_test.go

```diff
@@ -225,6 +226,33 @@ func TestGetTask_userInputErrors(t *testing.T) {
 	}
 }
 
+func TestGetTask_expiredTasks(t *testing.T) {
+	tc, ctrl := newTestCommand(t)
+	defer ctrl.Finish()
+	command := NewTaskCommand(tc.Command())
+	result := []*models.TaskSummary{
+		{TaskID: "id3"},
+	}
+
+	tc.Resolver.EXPECT().
+		Resolve("task", "name").
+		Return([]string{"id3", "id4", "id5"}, nil)
+
+	tc.Client.EXPECT().
+		ListTasks().
+		Return(result, nil)
+
+	tc.Client.EXPECT().
+		GetTask(gomock.Any()).
+		Return(&models.Task{}, nil).
+		Times(1) //only task 'id3' should result in a gettask call
```

Comment:
I like the comment, but the `Times(1)` is actually redundant with the `gomock` framework (you need to explicitly put `AnyTimes()` to allow the behavior you're looking to avoid)